scripts/update-setup-cli-dependencies.sh

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
```

Comment:
Was this script supposed to be named just `setup-cli-dependencies` or is the `update-` prefix deliberate?

Reply:
So, I am grouping the scripts used in the update process so that in a future PR, they can be separated into new folders and organized better.